CMakeLists.txt

```diff
@@ -133,14 +133,15 @@ SET (ORION_LIBS
     rest
     jsonParse
     jsonParseV2
-    apiTypesV2
+    rest
```

Comment:
Also, if the name of the function that is causing the problem is mentioned, then it will be easier to fix the problem later (assuming we have a case of cross dependencies in libraries)